pkg/controller/kieapp/webconsole.go

```diff
@@ -0,0 +1,99 @@
+package kieapp
+
+//go:generate go run -mod=vendor ./defaults/.packr/packr.go
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/shared"
+	"strings"
+
+	api "github.com/kiegroup/kie-cloud-operator/pkg/apis/app/v2"
+)
+
+func createConsoleYAMLSamples(r *Reconciler) {
+	log.Info("Loading CR yaml samples.")
+	contents, names, err := loadFiles("cryamlsamples")
+	if err != nil {
+		log.Error(err, "Failed to load cryamlsamples")
+		return
+	}
+
+	resMap, err := applyMultipleWebConsoleYamls(contents, r)
+	if err != nil {
+		log.Error(err, "cryamlsamples not successfully applied")
+		return
+	}
+
+	for k, v := range resMap {
+		log.Infof("yaml file: %s %s ", names[k], v)
+	}
+}
+
+func applyMultipleWebConsoleYamls(yamlString []string, r *Reconciler) (map[int]string, error) {
+	if yamlString == nil || len(yamlString) == 0 {
+		return nil, errors.New("Empty yaml list")
+	}
+	resMap := make(map[int]string)
+	for idx, yamlStr := range yamlString {
+		_, err := create(yamlStr, r)
+		if err != nil {
+			resMap[idx] = err.Error()
+		} else {
+			resMap[idx] = "Applied"
+		}
+	}
+	return resMap, nil
+}
+
+func create(yamlStr string, r *Reconciler) (bool, error) {
+	kieApp := api.KieApp{}
+	err := yaml.Unmarshal([]byte(yamlStr), &kieApp)
+	if err != nil {
+		return false, err
+	}
+
+	yamlSample, err := shared.GetConsoleYAMLSample(&kieApp)
+	if err != nil {
+		return false, err
+	}
+	err = r.Service.Create(context.TODO(), yamlSample)
+	if err != nil {
+		return false, err
+	}
+	return true, nil
+}
+
+func loadFiles(boxname string, path ...string) ([]string, []string, error) {
```

Comment:
Why is a boxname needed? Why not just hardcode to something on line 75?

Reply:
that way looks better, modified.